keyboards/satan/keymaps/vandheer/keymap_template.c

```diff
@@ -0,0 +1,21 @@
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  /* Keymap _BLANK
+   * ,-----------------------------------------------------------.
+   * |    |   |   |   |   |   |   |   |   |   |   |   |   |      |
+   * |-----------------------------------------------------------|
+   * |     |   |   |   |   |   |   |   |   |   |   |   |   |     |
+   * |-----------------------------------------------------------|
+   * |Nav~Esc|   |   |   |   |   |   |   |   |   |   |   |       |
+   * |-----------------------------------------------------------|
+   * |Shift   |   |   |   |   |   |   |   |   |   |   |          |
+   * |-----------------------------------------------------------|
+   * |Ctrl|Alt |Gui |                        |Gui |Alt |Fn |Macro|
+   * `-----------------------------------------------------------'
+   */
+[_BLANK] = KEYMAP_ANSI(
```

Comment:
```suggestion
[_BLANK] = LAYOUT_ansi(
```

Reply:
Actually, this should be:

```suggestion
[_BLANK] = LAYOUT_60_ansi(
```